packages/zk-eddsa-event-ticket-pcd/artifacts-unsafe/eddsaEventTicket.circom

```diff
@@ -57,6 +57,9 @@ template EdDSATicketToEventsPCD (nEvents) {
     signal input ticketIsRevoked;
     signal input revealTicketIsRevoked;
 
+    signal input ticketCategory;
```

Comment:
You need to regenerate the artifacts for this circuit as well, like you did for the other one.  They're separate scripts.